src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="#"
+    class="search"
+    method="get"
+    data-qa="big"
+    >
+      <label>
+        <img
+        src="images\Search.svg"
+        alt="icon"
+        class="icon">
+        <input
+        type="text"
+        class="input"
+        name="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
```

Comment:
check style guideline 
[CHECKLIST](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
```suggestion
    <form
      action="#"
      class="search"
      method="get"
      data-qa="big"
    >
      <label>
        <img
          src="images\Search.svg"
          alt="icon"
          class="icon"
        >
        
        <input
          type="text"
          class="input"
          name="search"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```